tests/lib/cisco_test.py

```diff
@@ -798,5 +807,15 @@ def testProtoInts(self):
     self.failUnless('permit udp any any range 1024 65535' in str(acl),
                     str(acl))
 
+  def testFragments(self):
+    self.naming.GetNetAddr.return_value = [nacaddr.IP('10.0.0.0/24')]
+    acl = cisco.Cisco(policy.ParsePolicy(GOOD_HEADER + GOOD_TERM_20,
+                                         self.naming), EXP_INFO)
+    first_string = 'permit ip 10.0.0.0 0.0.0.255 10.0.0.0 0.0.0.255 fragments'
```

Comment:
instead of first_string lets rename this to "expected"

Reply:
changed